Resources/Prototypes/Entities/Mobs/Player/observer.yml

```
@@ -71,6 +71,19 @@
   id: MobObserver
   components:
   - type: Spectral
+  - type: IntrinsicUI # Mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)